node-tests/test.js

```diff
@@ -257,4 +257,26 @@ describe('included()', function() {
       expect(cacheKey).to.equal(expectedCacheKey);
     });
   });
+
+  describe('duplicate plugin', function() {
+    let plugin = {
+      name: 'some-parallel-plugin',
+      plugin: 'some object',
+      baseDir: testBaseDir,
+      parallelBabel: dependentParallelInfo,
+    };
+
+    beforeEach(function() {
+      InlinePrecompile.parent.options.babel.plugins = [plugin];
```

Comment:
Can we change this test to be more "real world"? For example, I think it would be completely reasonable to invoke `.included()` directly twice and confirm we don't get two copies...